mmengine/hub/hub.py

```diff
@@ -80,4 +80,6 @@ def get_model(cfg_path: str, pretrained: bool = False, **kwargs):
         model = MODELS.build(cfg.model, default_args=kwargs)
         if pretrained:
             load_checkpoint(model, cfg.model_path)
+            # Hack to use pretrained weights
```

Comment:
The comments should be more detailed: "If we do not set `_is_init` here, Runner will call self.model.init_weights to overwrite the pretrained model"

Reply:
Good points! Fixed